transports/transport-rest/transport-rest-vertx/src/main/java/org/apache/servicecomb/transport/rest/vertx/RestServerVerticle.java

```diff
@@ -127,7 +127,9 @@ private HttpServerOptions createDefaultHttpServerOptions() {
     serverOptions.setIdleTimeout(TransportConfig.getConnectionIdleTimeoutInSeconds());
     serverOptions.setCompressionSupported(TransportConfig.getCompressed());
     serverOptions.setMaxHeaderSize(TransportConfig.getMaxHeaderSize());
-
+    if (endpointObject.isHttp2Enabled()) {
```

Comment:
this means h2 mode?
but how h2c can work?

Reply:
for h2 mode we need to enable ALPN
but for h2c(with prior knowledge) no change is required on the server side

Reply:
Can we set serverOptions.setUseAlpn(true) always?  See comments for protocol=h2.